variables.tf

```diff
@@ -41,7 +47,7 @@ variable "dependency_ids" {
 #######################
 
 variable "thanos" {
-  description = "Thanos settings"
+  description = "Most frequently used Thanos settings. This variable is merged with the local value `thanos_defaults`, which contains some sensible defaults. You can check the default values on the link:./local.tf[`local.tf`] file. If there still is anything other that needs to be customized, you can always pass on configuration values using the variable `helm_values`."
```

Comment:
What does "Most frequently" means here?
Do we really need this considering we have `helm_values` to pass values ? 
If this is a super set of Helm values maybe we can explode the map into different module variables.